### DISCUSSION TITLE
Editing the Web GUI of tasmota

### DISCUSSION THREAD

**Discussion by zeebak** [arendst/Tasmota#10901] [Q&A] [answered]
I want to change the web ui of the firmware. Maybe change some text and add/remove few buttons in gui

**Comment by sfromis** [comment, marked as answer]
As Tasmota is meant to be used via MQTT, standard setup includes no features for editing the web UI, outside of a very few selections. Using the scripting language you may be able to do something, but the web UI is really meant for admin/test, not for a making a user interface for daily operation.

**Comment by supperka** [comment]
only what can do , is change names of the relays by console cmd : webbutton(X) (text) 
X = number of relay
Text your text

and colors of  web UI

**Comment by meingraham** [comment]
With Tasmota scripting you can add/change objects on the web UI.
